### ISSUE TITLE
Cannot import numpy 1.8.1

### ISSUE THREAD

**Issue by almarklein** [ContinuumIO/anaconda-issues#104]
Windows 7 64, but using 32bit Python.

    conda                     3.4.0                    py33_0  
    numba                     0.13.0               np18py33_0  
    numpy                     1.8.1                    py33_0  
    python                    3.3.2                         0  

Log:

    In [1]: import numpy as np
    ---------------------------------------------------------------------------
    AttributeError                            Traceback (most recent call last)
    <ipython-input-1-4ee716103900> in <module>()
    ----> 1 import numpy as np
    
    C:\pyzo2013c_32\lib\site-packages\numpy\__init__.py in <module>()
        166         return loader(*packages, **options)
        167 
    --> 168     from . import add_newdocs
        169     __all__ = ['add_newdocs', 'ModuleDeprecationWarning']
        170 
    
    C:\pyzo2013c_32\lib\site-packages\numpy\add_newdocs.py in <module>()
        11 from __future__ import division, absolute_import, print_function
        12 
    ---> 13 from numpy.lib import add_newdoc
        14 
        15 ###############################################################################
    
    C:\pyzo2013c_32\lib\site-packages\numpy\lib\__init__.py in <module>()
        6 from numpy.version import version as __version__
        7 
    ----> 8 from .type_check import *
        9 from .index_tricks import *
        10 from .function_base import *
    
    C:\pyzo2013c_32\lib\site-packages\numpy\lib\type_check.py in <module>()
        9            'common_type']
        10 
    ---> 11 import numpy.core.numeric as _nx
        12 from numpy.core.numeric import asarray, asanyarray, array, isnan, \
        13                 obj2sctype, zeros
    
    C:\pyzo2013c_32\lib\site-packages\numpy\core\__init__.py in <module>()
        9 from . import numerictypes as nt
        10 multiarray.set_typeDict(nt.sctypeDict)
    ---> 11 from . import numeric
        12 from .numeric import *
        13 from . import fromnumeric
    
    C:\pyzo2013c_32\lib\site-packages\numpy\core\numeric.py in <module>()
        372 fromfile = multiarray.fromfile
        373 frombuffer = multiarray.frombuffer
    --> 374 may_share_memory = multiarray.may_share_memory
        375 if sys.version_info[0] < 3:
        376     newbuffer = multiarray.newbuffer
    
    AttributeError: 'module' object has no attribute 'may_share_memory'

**Comment by ilanschnell** [comment]
Is this still an issue?  Importing `numpy` is part of the regular test suite.

**Comment by almarklein** [comment]
No, I have not seen this later. I think this was due to a broken numpy, it looks similar to errors that I got because of that: when you `conda update numpy` on Windows, while numpy is imported somewhere (i.e. some dll's are loaded) the install may fail to update these dll's, but there is no warning about this.

The problem that numpy was imported when running `conda update numpy` was run was an error on my part, but maybe conda should not silently fail an install.
